homeassistant/components/recollect_waste/sensor.py

```diff
@@ -58,7 +58,7 @@ async def async_setup_platform(
     config: dict,
     async_add_entities: AddEntitiesCallback,
     discovery_info: dict = None,
```

Comment:
```suggestion
    config: ConfigType,
    async_add_entities: AddEntitiesCallback,
    discovery_info: DiscoveryInfoType | None = None,
  ```
  from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType